atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleOverflowWriteTable.java

```diff
@@ -368,8 +369,9 @@ private void deleteOverflow(String overflowTable, List<Object[]> args) {
     private void deleteOverflowRange(String overflowTable, String shortTableName, RangeRequest range) {
         StringBuilder query = new StringBuilder();
         query.append(" /* DELETE_RANGE_OVERFLOW (").append(overflowTable).append(") */ ");
-        query.append(" DELETE /*+ INDEX(m pk_").append(overflowTable).append(") */ ");
-        query.append("   FROM ").append(overflowTable).append(" m ");
+        query.append(" DELETE /*+ INDEX(m pk_").append(overflowTable).append(") ");
```

Comment:
why does this one just append the overflowTable name instead of using `PrimaryKeyConstraintNames.get`?

Reply:
I think the behavior is equivalent, but it's just weird that it is inconsistent.

Reply:
No good reason I can see, I wanted to make the smallest possible change, and left this inconsistency as is. TBH all this code could be cleaned up and the overflow migration parts stripped out.

Reply:
You'd have to ask your 2016 self :)